libraries/debug-certificate-manager/src/certificates.ts

```diff
@@ -16,28 +11,41 @@ import { runSudoSync, ISudoSyncResult } from './sudoSync';
 import { CertificateStore } from './CertificateStore';
 
 const serialNumber: string = '731c321744e34650a202e3ef91c3c1b9';
-const friendlyName: string = 'gulp-core-build-serve Development Certificate';
+const friendlyName: string = 'debug-certificate-manager Development Certificate';
 const macKeychain: string = '/Library/Keychains/System.keychain';
 
-let _certutilExePath: string;
+let _certutilExePath: string | undefined;
 
+/**
+ * The interface for a debug certificate instance
+ *
+ * @public
+ */
 export interface ICertificate {
-  pemCertificate: string;
-  pemKey: string;
+  /**
+   * Generated pem certificate contents
+   */
+  pemCertificate: string | undefined;
+
+  /**
+   * Private key used to sign the pem certificate
+   */
+  pemKey: string | undefined;
 }
 
 function _createDevelopmentCertificate(): ICertificate {
```

Comment:
Can you wrap all of these in a class?

Reply:
Let me know if this looks good. I called it, `CertificateManger`.